FirstAnalysis/trackingstudies.py

```diff
@@ -136,42 +78,38 @@ def tracking_resolution(var1, mode, var2, var3):
 
 
 def vertex_distributions(var):
-    # x, y, z
     fileo2 = TFile("../codeHF/AnalysisResults_O2.root")
     cvertex = TCanvas("cvertex", "vertex ")
     cvertex.SetCanvasSize(1500, 700)
     cvertex.Divide(2, 1)
+    cvertex_res = TCanvas("cvertex", "vertex ")
+    cvertex_res.SetCanvasSize(1500, 700)
+    cvertex_res.Divide(2, 1)
```

Comment:
there are only two plots totally, in line 84, the canvas has already be divided into 2.